helm/vitalam-node/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: node
+description: A Helm chart to deploy Substrate/Polkadot nodes
+type: application
+version: 0.23.0
+appVersion: "0.0.1"
```

Comment:
I think we should probably update these. Note that the helm version is now decoupled from the app version so that should also be the case here right?

Reply:
This is updated from the release that we forked from.
https://github.com/paritytech/helm-charts/blob/main/charts/node/Chart.yaml

Reply:
Hang on so the argument here is to not make this a `vitalam-node` chart but a generic substrate one?

Reply:
I assumed given the chart is a fork we increment from the currently released versions, rather than start our own versioning.

Reply:
Alright seems fair